### ISSUE TITLE
dataclasses missing in the deps?

### ISSUE THREAD

**Issue by soichih** [mxmlnkn/ratarmount#67]
Just FYI to @mxmlnkn 

I was noticing this error message after upgrading to the latest version (0.8.0) on Ubuntu 18.04 (using pip3)

```
Jul 11 16:44:02 wrangler2 automount[25310]: >>   File "/usr/local/bin/ratarmount", line 7, in <module>
Jul 11 16:44:02 wrangler2 automount[25310]: >>     from ratarmount import cli
Jul 11 16:44:02 wrangler2 automount[25310]: >>   File "/usr/local/lib/python3.6/dist-packages/ratarmount.py", line 21, in <module>
Jul 11 16:44:02 wrangler2 automount[25310]: >>     from dataclasses import dataclass
Jul 11 16:44:02 wrangler2 automount[25310]: >> ModuleNotFoundError: No module named 'dataclasses'
```

I just manually `pip3 install dataclasses` and fixed the issue.

**Comment by mxmlnkn** [comment]
Thanks for the report. The module only seems to be necessary for Python 3.6 (which is EOL in 5 months). I'm not sure why it worked in the CI, it was probably preinstalled or installed by some other Python package.

**Comment by mxmlnkn** [comment]
Hopefully fixed in 0.8.1
